### DISCUSSION TITLE
What is the name aria2 standing for?

### DISCUSSION THREAD

**Discussion by Nocheinstein** [aria2/aria2#2211] [General]
Is there also a downloader named Aria? Thanks!

**Comment by anderlli0053** [comment]
@Nocheinstein It is not an acronym, at least i don't think so, aria/aria2 does not stand for anything, the developer(s) just gave it that kind of name. "Aria" is a female name and also a musical term along other things.

aria is apparently predecessor to aria2, now obsolete project, but i cannot find any links or anything regarding the old downloader.

**aria**: a chosen project name, possibly inspired by the musical term aria or female name (Aria)

**2**: part of the project’s name, likely distinguishing it from an earlier aria downloader/predecessor

**aria2c**: the command-line executable; the "c" conventionally indicates the "command-line" client

In other cases ARIA can stand or "Advanced Rich Internet Application", but i assume that this is not the case here.

This is my take on it, since these information cannot be found anywhere, not even here in this repository.
